apps/wallet-mobile/src/components/ProgressStep.tsx

```diff
@@ -34,27 +38,32 @@ export const ProgressStep = ({currentStep, totalSteps, displayStepNumber}: Progr
   return <View style={[styles.bar]}>{steps}</View>
 }
 
-const styles = StyleSheet.create({
-  bar: {
-    backgroundColor: COLORS.WHITE,
-    height: 10,
-    flexDirection: 'row',
-  },
-  step: {
-    backgroundColor: COLORS.LIGHT_POSITIVE_GREEN,
-    alignItems: 'center',
-    justifyContent: 'center',
-    flexGrow: 1,
-  },
-  markedStep: {
-    marginHorizontal: 0.5,
-  },
-  todoStep: {
-    backgroundColor: '#C9EDE5',
-  },
-  stepNumber: {
-    fontSize: 7,
-    lineHeight: 10,
-    color: COLORS.WHITE,
-  },
-})
+const useStyles = () => {
+  const {theme} = useTheme()
+  const {color} = theme
+  const styles = StyleSheet.create({
+    bar: {
+      backgroundColor: color.gray.min,
+      height: 10,
+      flexDirection: 'row',
+    },
+    step: {
+      backgroundColor: color.secondary[400],
+      alignItems: 'center',
+      justifyContent: 'center',
+      flexGrow: 1,
+    },
+    markedStep: {
+      marginHorizontal: 0.5,
+    },
+    todoStep: {
+      backgroundColor: color.secondary[200],
+    },
+    stepNumber: {
+      fontSize: 7,
```

Comment:
@SorinC6 can you check with Design, whether it has typography or not, it **must** fall into one of the typography options, there will be **NO** exceptions.